app/views/layouts/application.html.erb

```diff
@@ -12,6 +12,8 @@
 
     <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
     <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
+    <%= javascript_include_tag 'https://js.stripe.com/v3/', 'data-turbolinks-track': 'reload' %>
+

```

Comment:
can this be moved to the application.js file somehow?

Reply:
I don't know

Reply:
I thought you can use `yarn add` for that

Reply:
it didn't work